### ISSUE TITLE
Seems almost none of the example pool links work

### ISSUE THREAD

**Issue by fossum** [bonesoul/CoiniumServ#949]
I was just on the [example pools page](https://github.com/bonesoul/CoiniumServ/wiki/Pools) and it turns out that all but iSpace is broken.
If you want to interest people in your server, I'd suggest sending them to working links.

**Comment by smartmanjp** [comment]
you are right I paid for this software and i try to make it work on Windows 10 and Linux not to good
the instruction do not  work so what is going on can you make step by step for windows and Linux for Paid software please

**Comment by jeong760** [comment]
@fossum, you can find a advence.json on CoimiumServ release 1.5. and if you needed I will help to explanation how to.. actually I purchased a paid version but free also enough for Linux and Windows.

**Comment by jimoquinn** [comment]
Because keeping a pool running reliably can be real difficult, especially if you're not a coder, sys-admin, and familiar with the guts of the coin.  Most miners are not loyal, so taking the pool offline for a few hours, not to mention 24-36 hours, could result in a mass exodus.

I'd bet more pools are birthed and die in a month than exist total.  The real pool killer is when pool patches or coin code patches need to be applied with almost no downtime.

**Comment by smartmanjp** [comment]
Any in here can explaim to me how to step by step how to add coin and get to admin panel o this server please
